src/tests/App.test.js

```diff
@@ -11,4 +11,19 @@ describe('<App />', () => {
     const div = document.createElement('div');
     ReactDOM.render(<App />, div);
   });
+
+  it("renders one display component", () => {
```

Comment:
Granted this test could have been expanded to check for every single component inside the app, i am glad you checked to make sure it only contains one display. Our calculator would look strange with 2 of them 👍